groups/paths.yaml

```
@@ -345,6 +349,11 @@ DeviceDelete:
 
     A gratuitous update of the corresponding Group endpoint will occur
     to update clients as to the new devices, actions, state, and properties.
+
+    When the last device is removed (DELETEd) from a group shard,
+    the group shard will vanish.
```

Maybe it is good to add the case where deleting the device from the Bond, also vanishes the group shard.
```suggestion
    When the last device of  a group shard is removed (DELETEd) from
    the Bond or from the group, the group shard will vanish.
```